### DISCUSSION TITLE
Disable LYWSD03MMC

### DISCUSSION THREAD

**Discussion by darkman7hr** [arendst/Tasmota#20972] [Q&A]
Is there any way to disable LYWSD03MMC entirely?
In documentation there is "NRFUse", but it's not longer available.

**Comment by barbudor** [comment]
What problem are you looking to solve ?

**Comment by darkman7hr** [comment]
My ESP32-C3 Supermini is rebooting often, so I'm in process of diagnosing.

**Comment by barbudor** [comment]
Look at https://tasmota.github.io/docs/Commands/#ble-esp32

There's a mi32block command
Never used it but may suis your need
You'll have to block them 1 by 1

**Comment by darkman7hr** [comment]
I saw that, but scans will be made in background I guess. If I disable BLE, no reboots. I have one GAS sensor on pin analog 1 but his digital is connected to 5, maybe that's causing some issue.. 
Thank you anyhow.. maybe that NRFUse should be purged from code and documentation, it's confusing.

**Reply by sfromis** [reply]
The `NRFUse` command is in a section about devices based on using nRF24L01 chips, meaning that is is not relevant when your BLE setup is not using such a chip, but instead internal BLE support of ESP32.

**Comment by wjohn007** [comment]
I recently started using BLE. 
My own application (Berry-Script) was also started.
After that the controller was no longer accessible. 
Only when I connected it to the serial port did it become clear that the memory was too small.
 BLE requires a lot of it.
The BLE Gateway now runs exclusively on a controller, and everything has been stable since then.

John
